.buildkite/pipeline.provision.yml

```diff
@@ -67,6 +52,21 @@ steps:
 
   - wait
 
+  - label: ":pulumi: Preview grapl/integration-tests/testing environment changes in Staging account"
+    plugins:
+      - grapl-security/vault-login#v0.1.0
+      - grapl-security/vault-env#v0.1.0:
+          secrets:
+            - PULUMI_ACCESS_TOKEN
+      - grapl-security/pulumi#v0.1.2:
+          command: preview
+          project_dir: pulumi/integration_tests
+          stack: grapl/testing
+    agents:
+      queue: "pulumi-staging"
+
+  - wait
+
```

Comment:
🤔 Does previewing the integration tests provide much benefit?

Reply:
Maybe by making it clear where something fails? I'll keep it for now, but I'm open to removing it